acceptance/auth/bundle_and_profile/output.txt

```diff
@@ -19,7 +19,7 @@ Exit code: 1
 
 === Inside the bundle, target and not matching profile
 >>> errcode [CLI] current-user me -t dev -p profile_name
-Error: cannot resolve bundle auth configuration: config host mismatch: profile uses host https://non-existing-subdomain.databricks.com, but CLI configured to use [DATABRICKS_TARGET]
+Error: cannot resolve bundle auth configuration: the host in the profile (https://non-existing-subdomain.databricks.com) doesn’t match the host configured in the bundle ([DATABRICKS_TARGET]). Did you mean to use the profile DEFAULT?
```

Comment:
> Did you mean to use the profile DEFAULT?

Not super clear, does it mean passing "-p DEFAULT"? or omitting "-p" or something else?

Reply:
Can we elaborate this more?

> The profile "DEFAULT" has host="https://bla.com" that matches host in the bundle/dev target. To select it, pass "-t DEFAULT".